### ISSUE TITLE
Use hashes in generated file names

### ISSUE THREAD

**Issue by insin** [insin/nwb#38]
(SInce I had to advise someone in work how to force their browser to bypass the cache today)

**Comment by insin** [comment]
This depends on #34, as we'll need to generate HTML with the correct file names

**Comment by insin** [comment]
Using a cachebusting param seems like a better idea

**Comment by insin** [comment]
Closing in favour of #73
